data/drak.txt

```diff
@@ -168,9 +168,9 @@ effect "antimatter dust"
 
 
 
-ship "Archon"
+ship "Archon A"
```

Comment:
The ship name shouldn't have the variant letter in it, otherwise this actually shows up in game. (The world-ship a image doesn't go to the "World-Ship A" ship, it's just "World-Ship".)